source/administration/data-retention.rst

```diff
@@ -78,3 +78,11 @@ In v1.5 and later of the iOS and Android apps, messages and files are deleted fr
 2. When the user puts the app into the background.
 
 In v1.4 and earlier of the mobile apps, messages and files are not cleared from local storage when the data retention policy is enabled.
+
+How do I know if a data retention job fails?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost provides the status of each data retention job in **System Console** > **Compliance** > **Data Retention Policy**. Here, you can see if the job succeeded or failed, including XXX.
```

Comment:
Katie: wondering if you know what's included in the status table -- # of messages deleted, perhaps?

Reply:
Please add : "including the details of the error."  The grid includes Status, Finish Time, Run Time and Details.  Details is only populated when there is an error.